geode-core/src/test/java/org/apache/geode/test/dunit/rules/ServerStarterRule.java

```diff
@@ -145,6 +153,7 @@ public void startServer(boolean pdxPersistent) {
     CacheFactory cf = new CacheFactory(this.properties);
     cf.setPdxReadSerialized(pdxPersistent);
     cf.setPdxPersistent(pdxPersistent);
+    this.pdxPersistent = pdxPersistent;
```

Comment:
I don't think we should add this line here. 

we can get rid of this "public void startServer(boolean pdxPersistent)"  method now since we have withPdxPersistant added. the flag should solely be determined by the value of the member variable.